proposals/0029-cooperative-vector.md

```diff
@@ -84,10 +84,10 @@ this should give a sense of how these new operations can be used.
 > NOTE: see proposal [0031] for full details on the HLSL API.
 
 ```c++
-ByteAddressBuffer inputMatrix0; 
-ByteAddressBuffer inputMatrix1; 
-ByteAddressBuffer biasVector0; 
-ByteAddressBuffer biasVector1;
+ByteAddressBuffer inputMatrix0; // note read-only buffer
+ByteAddressBuffer inputMatrix1; // note read-only buffer
+ByteAddressBuffer biasVector0;  // note read-only buffer
+ByteAddressBuffer biasVector1;  // note read-only buffer
```

Comment:
I think one comment at the top of the block pointing out that they are all read-only buffers is sufficient.